fuzz/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "ruff-fuzz"
+version = "0.0.0"
+authors = [
+  "Addison Crump <research@addisoncrump.info>",
+]
+publish = false
+edition = "2021"
+
+[features]
+full-idempotency = []
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+arbitrary = { version = "1.3.0", features = ["derive"] }
+libfuzzer-sys = { version = "0.4.1" }
+ruff = { path = "../crates/ruff" }
+ruff_python_ast = { path = "../crates/ruff_python_ast" }
+ruff_python_formatter = { path = "../crates/ruff_python_formatter" }
+rustpython-ast = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "7a3eedbf6fb4ea7068a1bf7fe0e97e963ea95ffd", default-features = false, features = ["all-nodes-with-ranges"]}
+rustpython-parser = { git = "https://github.com/astral-sh/RustPython-Parser.git", rev = "7a3eedbf6fb4ea7068a1bf7fe0e97e963ea95ffd", default-features = false, features = ["full-lexer", "all-nodes-with-ranges"] }
```

Comment:
I see. This is not part of the workspace. What's the motivation for not making this crate part of the Ruff workspace? It being its own workspace increases makes versioning harder (we now need to keep in mind to update rust python here and in the main ruff workspace)

Reply:
See previous answer :slightly_smiling_face:

Reply:
Should be able to, yes -- let me try this.

Reply:
Turns out I didn't even use these deps :upside_down_face: